### PR TITLE
Adding missing desc

### PR DESCRIPTION
**Pull request by get-itips**
Closes https://github.com/MicrosoftDocs/office-docs-powershell/issues/7044

@tseward do you happen to know more details about the other parameters that have PARAMVALUE as a description? `-EncryptionMetaInfo` and `-MigrationSourceLocations` ? Thanks

### CODE REVIEW

**Comment by tseward** [comment]
`MigrationFileEncryptionInfo` is an object consisting of:

`[byte]InitializationVector, [string]FileName, [string]MD5Hash`

`MigrationPackageLocation` is an object consisting of:

`[DirectoryInfo]packageDirectory, [DirectoryInfo]filesDirectory`

**Comment by get-itips** [comment]
> `MigrationFileEncryptionInfo` is an object consisting of:
> 
> `[byte]InitializationVector, [string]FileName, [string]MD5Hash`
> 
> `MigrationPackageLocation` is an object consisting of:
> 
> `[DirectoryInfo]packageDirectory, [DirectoryInfo]filesDirectory`

Thanks @tseward we are closer.

@trent-green can you help me getting friendly descriptions for these parameters?

**Comment by get-itips** [comment]
@trent-green Can you help me getting the missing descriptions? I appreciate it.

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOMigrationPackageAzureSource.md [hunk 1]
```diff
@@ -196,7 +196,7 @@ Accept wildcard characters: False
 
 ### -MigrationSourceLocations
 
-PARAMVALUE: MigrationPackageLocation
+Possible Source locations to migrate
```
Comment (dariomws) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOMigrationPackageAzureSource.md:199]:
```suggestion
Possible Source locations to migrate.
```

**Review by dariomws — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Kateyanne

**Pull request merged**